sdk-cpp/src/internal/com/deliveryoptimization/do.hpp

```diff
@@ -80,10 +80,10 @@ typedef enum _DODownloadProperty
     DODownloadProperty_IntegrityCheckInfo,
     DODownloadProperty_IntegrityCheckMandatory,
     DODownloadProperty_TotalSizeBytes,
-    DODownloadProperty_DisallowOnCellular,                  // TODO: Document this (Win11)
```

Comment:
>DODownloadProperty_DisallowOnCellular [](http://example.com/codeflow?start=4&length=37)

It maybe useful to keep the min build required for these new properties, as comments. Win11/22000 for the first two and the last two requires pre-release builds prior to release of next Win11 version.

Reply:
I'll add a comment for the first two, I'm going to delete the last two for now. By the time these become available the windows sdk will have shipped with these two.